src/main/java/de/kekru/struktogrammeditor/control/GlobalSettings.java

```diff
@@ -142,14 +142,11 @@ private static void loadSettings () {
 		//Neue Einstellungsdatei einlesen
 		if(einstellungsDatei.exists()){
 			Properties pr = new Properties();
-			try {
-				BufferedInputStream in = new BufferedInputStream(new FileInputStream(einstellungsDatei));
+			try (BufferedInputStream in = new BufferedInputStream(new FileInputStream(einstellungsDatei))){
 				pr.load(in);
-				in.close();
 			} catch (IOException e) {
 				e.printStackTrace();

```

Comment:
Better:

 ```java
} catch (IOException e) {
    throw new RuntimeException(e);
}
```

Reply:
I don't know why I can't comment on the others so here is everything in one comment:
Struktogramm.java: Added only printToPng 
I'm deleting commented code as I come across it
And to my Formatter: Sorry I don't like the braces style used in the project but I don't want to do it on every line manually, I will fix every f*cked up comment as I come across

I've also reverted the github link but I still have problems with buildnumber-maven-plugin, Eclipse seems to hate it

Reply:
Ok, thanks. I'll try to check the whole changes on the weekend (too much to do at work at the moment)

Reply:
Have you already pushed your latest changes?
The latest commit (that I can see here) is 21 days ago

Reply:
I don't know why I can't comment on the others so here is everything in one comment:
Struktogramm.java: Added only printToPng 
I'm deleting commented code as I come across it
And to my Formatter: Sorry I don't like the braces style used in the project but I don't want to do it on every line manually, I will fix every f*cked up comment as I come across

I've also reverted the github link but I still have problems with buildnumber-maven-plugin, Eclipse seems to hate it

I now pushed the updated code, sorry for the delay